rj_gameplay/rj_gameplay/play/test_pass.py

```diff
@@ -43,7 +45,9 @@ def tick(
         skill_dict = {}
         if not self.pass_tactic.is_done(world_state):
             skills = self.pass_tactic.tick(role_results[self.pass_tactic], world_state)
+            skills += self.seek_tactic.tick(role_results[self.seek_tactic], world_state)
```

Comment:
IMO should have a line like
```
skills = []
```
before the if, then delete the else branch for clarity.